lib/data.js

```diff
@@ -19,14 +20,14 @@ function collectData(modelConfig) {
 }
 
 function getData(modelConfig, { rewriteCache, warmup, pretty } = {}) {
-    const cacheFile = getCacheFilename(modelConfig);
-
-    if (!cacheFile && warmup) {
```

Comment:
Because we don't know which cache file we need at this moment. The new one, or the latest of existed. But yes there is a mistake, should be `id (warmup) {...` Fixed it

Reply:
Wait a minute!

Reply:
Yeah you were right, I messed up with this conditions) returned all back.